mongo/README.md

```diff
@@ -116,15 +116,16 @@ db.createUser({
 })
 ```
 
-Then create the same user from a mongos proxy, this also has the side effect of creating the local user in the config servers and allows direct connection.
+Then create the same user from a mongos proxy. This action creates the local user in the config servers and allows direct connection.
 
-
-##### Configure the agents
+##### Configure the Agents
 1. Configure one Agent for each member of each shard.
 2. Configure one Agent for each member of the config servers.
-3. Configure one extra Agent to connect to the cluster through a mongos proxy. This mongos can be a new one dedicated to monitoring purposes or an existing one.
+3. Configure one extra Agent to connect to the cluster through a mongos proxy. This mongos proxy can be a new one dedicated to monitoring purposes, or an existing mongos proxy.
```

Comment:
Should `mongos` be capitalized? If yes, it's used additional times below.

Reply:
apparently `mongos proxy` is a thing in mongodb, it'll show errors like `No mongos proxy available`